### PR TITLE
feat: code review fixes

### PR DESCRIPTION
**Pull request by FrolYo**

### CODE REVIEW

File: src/dotnet/Host/Program.cs [hunk 1]
```diff
@@ -47,12 +47,12 @@ static void AdjustThreadPool()
             ThreadPool.GetMinThreads(out int currentMinWorker, out int currentMinIo);
             if (currentMinIo < minIoThreads) {
                 if (!ThreadPool.SetMinThreads(currentMinWorker, minIoThreads)) {
-                    throw new Exception("ERROR: Can't set min IO threads");
+                    throw new Exception("ERROR: Can't set min IO threads.");
```
Comment (alexyakunin) [inline src/dotnet/Host/Program.cs:50]: `Excepton` shouldn't be thrown.

File: src/dotnet/ClientApp/MauiProgram.cs [hunk 2]
```diff
@@ -63,7 +63,7 @@ public static MauiApp CreateMauiApp()
         //var settings = builder.Configuration.Get<ClientAppSettings>();
         var settings = new ClientAppSettings { BaseUri = GetBackendUrl() };
         if (string.IsNullOrWhiteSpace(settings.BaseUri))
-            throw new Exception("Wrong configuration, base uri can't be empty");
+            throw new Exception("Wrong configuration, base uri can't be empty.");
```
Comment (alexyakunin) [inline src/dotnet/ClientApp/MauiProgram.cs:66]: `Excepton` shouldn't be thrown.

File: src/dotnet/Users.Service/UserAvatars.cs [hunk 3]
```diff
@@ -68,12 +68,10 @@ public virtual async Task SetDefault(IUserAvatars.SetDefaultCommand command, Can
...
             var avatar = await _userAvatarsBackend.Get(avatarId, cancellationToken).ConfigureAwait(false);
             if (avatar == null || avatar.UserId != user.Id)
-                throw new InvalidOperationException("Invalid AvatarId");
+                throw new InvalidOperationException("Invalid AvatarId.");
         }
 
-        var userProfile = await _userProfilesBackend.Get(user.Id, cancellationToken).ConfigureAwait(false);
-        if (userProfile == null)
-            throw new InvalidOperationException("User profile does not exist");
+        var userProfile = await _userProfilesBackend.Get(user.Id, cancellationToken).Required().ConfigureAwait(false);
```
Comment (alexyakunin) [inline src/dotnet/Users.Service/UserAvatars.cs:74]: `Required` -> `AssertActive`? It's a front-end service.

**Review by alexyakunin — CHANGES_REQUESTED** [review]

File: src/dotnet/Users.Service/UserAvatars.cs (hunk 3 above)
Comment (alexis-kochetov) [reply src/dotnet/Users.Service/UserAvatars.cs:74]: Disagree, `Required` is a generic extension that works with all nullable results

**Review by alexis-kochetov — APPROVED** [review]

File: src/dotnet/Host/Program.cs (hunk 1 above)
Comment (FrolYo) [reply src/dotnet/Host/Program.cs:50]: fixed

File: src/dotnet/ClientApp/MauiProgram.cs (hunk 2 above)
Comment (FrolYo) [reply src/dotnet/ClientApp/MauiProgram.cs:70]: fixed

**Pull request merged**